Assets/MirageXR/Player/Scripts/Moodle/DBManager.cs

```diff
@@ -154,6 +162,23 @@ public static LearningRecordStores publicCurrentLearningRecordStore
             set => CurrentLearningRecordStore = value;
         }
 
+        public static ShowBy currentShowby
+        {
+            get => CurrentShowby;
+            set => CurrentShowby = value;
+        }
+
+        public static SortBy currentSortby
+        {
+            get => CurrentSortby;
+            set => CurrentSortby = value;
+        }
+
+        public static string Username
+        {
+            get => username;
+        }
+
```

Comment:
what's the point?

Reply:
My mistake, made this variable thinking I'd need it then didn't use it. Has been removed